.github/workflows/release.yaml

```diff
@@ -95,6 +95,14 @@ jobs:
           ./hack/docker-login.sh
           ./hack/release.sh
 
+      - name: scan source
+        run: |-
+          grype .
+
+      - name: scan image
+        run: |-
```

Comment:
differently from the `validation` workflow, here we don't need to point at a `DOCKER_CONFIG` because the images are all pushed to DockerHub, which is public (thus, anon auth is fine)

Reply:
Does this mean we can get rid of lines 61-62? Or is this no longer true and that is why it was marked resolved?